lib/parsers/ogg_parser.rb

```diff
@@ -0,0 +1,42 @@
+# https://xiph.org/vorbis/doc/Vorbis_I_spec.pdf
+# https://en.wikipedia.org/wiki/Ogg#Page_structure
+class FormatParser::OggParser
+  # The maximum size of an Ogg page is 65,307 bytes.
+  MAX_POSSIBLE_PAGE_SIZE = 65307
```

Comment:
That can be done later. At the moment we are quite liberal with cache page sizes since performing an HTTP request with latencies and resolves and all takes longer than simply slurping in more data. The page size is generous and you will be touching the last page anyway, it will be in memory - so no worries 👍